frontend/deploy/helm/frontend/templates/frontend.secret-refresher.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend-certificate-refresher
+  namespace: aks-istio-ingress
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: frontend-certificate-refresher
+  template:
+    metadata:
+      labels:
+        app: frontend-certificate-refresher
+    spec:
+      containers:
+        - command:
+            - "/bin/sleep"
```

Comment:
I find the name confusing, How is this refreshing anything?

Reply:
So what it actually does is because the secret is mounted from the keyVault (via the SecretProviderClass) it's watching the secret and if it changes it will update the secret. I get what you're saying about the naming though.... Would adding the above explanation to this yaml help?

Reply:
Done